### PR TITLE
Integration test readme update

### PR DESCRIPTION
**Pull request by dmcgowan**
Updates the readme to mention running the tests using golem.
Also provides instructions for making test development easier.

### CODE REVIEW

File: contrib/docker-integration/README.md [hunk 1]
```diff
@@ -1,136 +1,63 @@
...
+The integration tests configure components using docker compose
+(see docker-compose.yaml) and the runner can be using the golem
+configuration file (see golem.conf).
+
+## Running integration tests
+
+### Run using multiversion script
+
+The integration tests in the `contrib/docker-integration` directory can be simply
+run be executing the run script `./run_multiversion.sh`. If there is no running
```
Comment (RichardScothern) [inline contrib/docker-integration/README.md]: by executing

File: contrib/docker-integration/README.md [hunk 2]
```diff
@@ -1,136 +1,63 @@
...
+
+`golem -i golem-dind:latest,docker:1.10.3-dind,1.10.3 -i golem-distribution:latest,registry:2.2.1 .`
+
+
+#### Use golem caching for developing tests
+
+Golem allows caching image configuration to reduce test start up time.
+Using this cache will allow tests with the same set of images to start
+up quickly. This can be useful when developing tests and needing the
+test to run quickly. If changes are which effect the image (such as
```
Comment (RichardScothern) [inline contrib/docker-integration/README.md]: remove 'are'

**Comment by RichardScothern** [comment]
LGTM (nits aside)

File: contrib/docker-integration/README.md (hunk 2 above)
Comment (dmcgowan) [inline contrib/docker-integration/README.md]: Updated to "If there are changes", removing the "are" was not enough to make the sentence correct

**Pull request merged**